packages/bpk-docs/src/components/neo/DocsPageWrapper/DocsPageWrapper.js

```diff
@@ -0,0 +1,126 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import BpkContentContainer from 'bpk-component-content-container';
+import BpkHorizontalNav, {
+  BpkHorizontalNavItem,
+} from 'bpk-component-horizontal-nav';
+import BpkSmallMobileIcon from 'bpk-component-icon/sm/mobile';
+import BpkSmallWindowIcon from 'bpk-component-icon/sm/window';
+import { withButtonAlignment } from 'bpk-component-icon';
+import { cssModules } from 'bpk-react-utils';
+import isString from 'lodash/isString';
+
+import Heading from './../Heading';
+
+import STYLES from './DocsPageWrapper.scss';
+import Paragraph from '../Paragraph';
+
+const getClassName = cssModules(STYLES);
+
+const AlignedBpkSmallMobileIcon = withButtonAlignment(BpkSmallMobileIcon);
```

Comment:
Don't these api's need a 2nd and 3rd argument? All the examples on this page do: https://backpack.github.io/components/utilities/alignment

Reply:
I may be wrong but I don't think we do. The "Icon alignment within a button" on the page you linked to doesn't use a second or third argument, which is what I based this off.

Reply:
Ah your correct - `withButtonAlignment` is an alias around `withAlignment` with the lineHeight and icon size already set.